dashboard/app/models/user.rb

```diff
@@ -683,6 +686,16 @@ def self.find_by_hashed_email(hashed_email)
     migrated_user || User.find_by(hashed_email: hashed_email)
   end
 
+  def primary_contact_info
+    # TODO: Make sure this works?
+    return @primary_contact_info unless @primary_contact_info.nil?
```

Comment:
So in doing some testing with pry in a unit test, I think this is working? If there is no email set on any of the authentication options, and you run `user.primary_contact_info`, it returns nil. When you add an email to an authentication_option and rerun the command, it will return that authentication option.

Reply:
Refactored this to work and not break existing functionality.